openedx/core/djangoapps/user_api/api.py

```diff
@@ -278,15 +278,10 @@ def _add_email_field(self, form_desc, required=True):
         # meant to hold the user's email address.
         email_label = _(u"Email")
 
-        # Translators: This example email address is used as a placeholder in
-        # a field on the registration form meant to hold the user's email address.
-        email_placeholder = _(u"username@domain.com")
-
         form_desc.add_field(
             "email",
             field_type="email",
-            label=email_label,
-            placeholder=email_placeholder,
+            placeholder=email_label,
```

Comment:
this is somewhat different from the html that the js in this repo expects https://github.com/zellwk/adaptive-placeholders
but if for whatever reason we don't end up using that repo, jae's approach for the test didn't require changing the html

Reply:
I was slightly hesitant about using that repo, because it expects us to use normalize.css, which I don't believe we do, and there's a process for getting clearance from the open source team (though with an MIT license, that should be fine).

It's a good call to keep them as labels, though. I forgot about the a11y implications of removing them.

Reply:
in that case we can use the css/js Jae wrote in his test for the labels both above and below the field
changing their position is the only way to implement that behavior anyway